docs/utilities/oredictionary.md

```diff
@@ -0,0 +1,77 @@
+OreDictionary
+======
+
+The OreDictionary exists primarily for inter-mod compatibility. 
+
+By registering ores to the OreDictionary, mods make it possible to use the ores and ingots from another, similar mod in their recipes.
+
+Despite its name, the OreDictionary is used for much more than ores. Any item that is similar to another item (for example, dyes) can be registered to and used with the OreDictionary.
+
+## OreDictionary Name Convention
```

Comment:
This does not follow the standards in the guidelines. Secondary headers should use the underline syntax. See http://mcforge.readthedocs.io/en/latest/styleguide/